tests/TestBatch/test_ip_configure_load.tcl

```diff
@@ -0,0 +1,56 @@
+#Copyright 2022 The Foedag team
+
+#GPL License
+
+#Copyright (c) 2022 The Open-Source FPGA Foundation
+
+#This program is free software: you can redistribute it and/or modify
+#it under the terms of the GNU General Public License as published by
+#the Free Software Foundation, either version 3 of the License, or
+#(at your option) any later version.
+
+#This program is distributed in the hope that it will be useful,
+#but WITHOUT ANY WARRANTY; without even the implied warranty of
+#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#GNU General Public License for more details.
+
+#You should have received a copy of the GNU General Public License
+#along with this program.  If not, see <http://www.gnu.org/licenses/>.
+######################################################################
+
+# This is a test for GEMINIEDA-352
+# This test ensures that the IP_Catalog is automatically loaded when a user
+# runs a script with configure_ip, but doesn't bother to call add_litex_ip_catalog
+
+set platform $::tcl_platform(platform)
```

Comment:
Can we add a TODO here to fix these with the GH-# ?

Reply:
done